test/run_test.go

```diff
@@ -137,7 +137,7 @@ func TestCgoOk(t *testing.T) {
 			"--timeout=3m",
 			"--enable-all",
 			"-D",
-			"nosnakecase,gci",
+			"nosnakecase,gci,gofactory",
```

Comment:
I hope that is correct solution to pass tests.

Reply:
your tests are not in the right places.

Reply:
Do you mean I should move `test/testdata/gofactory/blocked.go` to `test/testdata/gofactory/gofactory_package_globs_only.go`?

Reply:
no, your tests must be inside `test/testdata/` and not a dedicated folder.
and your linter should be removed from `test/linters_test.go`.